packages/safe-apps-test-app/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@gnosis.pm/safe-apps-test-app",
```

Comment:
If we remove @gnosis.pm then we can `yarn workspace safe-apps-test-app start` without the prefix

Reply:
currently all the packages follow the same convention `@gnosis.pm/package-name` so let's keep it until we get a new npm org name

Reply:
ok!